hooks/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+
+cd $SCRIPT_DIR/../lambda/custom
+yarn build
+yarn zip
```

Comment:
If this script is present, it is executed when running ```ask deploy``` and it expects a ```build.zip``` artifact to be created.  If this script doesn't create it, ```ask deploy``` will fail.